React hook movie list/src/App.js

```diff
@@ -0,0 +1,76 @@
+import './App.css';
+import React, { useState }  from 'react';
+import { useContext } from 'react';
+import { Button, Modal } from 'antd';
+import { ThemeContext } from "./Theme";
+
+function App() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const { theme,toggleTheme  } = useContext(ThemeContext);
+  const showModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleOk = () => {
+    setIsModalOpen(false);
+  };
+
+  const handleCancel = () => {
+    setIsModalOpen(false);
+  };
+
+  return (
+    <div className={`App ${theme}`}>
+      <button onClick={() => toggleTheme()}>{theme}</button>
+      <h1>Movie List</h1>
+      <table>
```

Comment:
this whole thing should be loaded from a API or from a JSON config. cannot be hardcoded

Reply:
Input is loaded from json file